pkg/liqonet/iptables/iptables.go

```diff
@@ -570,6 +641,68 @@ func (h IPTHandler) insertRulesIfNotPresent(table, chain string, rules []string)
 	return nil
 }
 
+func getPostroutingRules(isGateway bool, tep *netv1alpha1.TunnelEndpoint) ([]string, error) {
+	clusterID := tep.Spec.ClusterID
```

Comment:
Can this parameter check be factorized with getPreRoutingRules with a common method?